CMakeLists.txt

```diff
@@ -49,11 +50,8 @@ endif(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 # rocAL Default Options
 option(ENHANCED_MESSAGE "rocAL Enhanced Message Option"        ON)
 option(GPU_SUPPORT      "Build rocAL with GPU Support"         ON)
-option(BUILD_DEV        "Build rocAL Development Package"      ON)
-option(AMD_FP16_SUPPORT "Build rocAL with float16 Support"    OFF)
 option(BUILD_PYPACKAGE  "Build rocAL Python Package"           ON)
-option(BUILD_WITH_AMD_ADVANCE "Build rocAL for advanced AMD GPU Architecture"    OFF)
-option(PYTHONVERSION    "Python version to build rocal" "")
+option(BUILD_WITH_AMD_ADVANCE  "Build rocAL with Advanced GPU support" OFF)
```

Comment:
@hansely why was pythonversion removed? Seems like you have not synced with ToT.